src/main/java/com/stuypulse/robot/subsystems/amper/Amper.java

```diff
@@ -90,6 +92,21 @@ public final boolean isAtTargetHeight(double epsilonMeters) {
         return Math.abs(getTargetHeight() - getLiftHeight()) < epsilonMeters;
     }
 
+    public final void setMinHeight(double height) {
+        minHeight = Math.max(Settings.Amper.Lift.MIN_HEIGHT, height);
+    }
+
+    public final double getMinHeight() {
+        return minHeight;
+    }
+
+    public void setSafeHeight() {
+        if(getLiftHeight() < Settings.Amper.Lift.UNSAFE_CLIMB_HEIGHT) {
+            setTargetHeight(Settings.Amper.Lift.SAFE_CLIMB_HEIGHT);
+            setMinHeight(Settings.Amper.Lift.SAFE_CLIMB_HEIGHT);
+        }
+    }
```

Comment:
The unsafe climb height should be checking the climber, not the lift -- but I think for now maybe we just check if the climber is going down fast like how we did at FLR
Put the setTargetHeight and setMinHeight [here](https://github.com/StuyPulse/Izzi/blob/0b08c5a05f265ba55074ccfb225f341cfa5435df/src/main/java/com/stuypulse/robot/commands/climber/ClimberDrive.java#L54) -- target height is already set here